TeachingRecordSystem/tests/TeachingRecordSystem.Core.Tests/Jobs/InductionStatusUpdatedSupportJobTests.cs

```diff
@@ -0,0 +1,267 @@
+        jobMetaData.Metadata = new Dictionary<string, object>
+        {
+            { "LastRunDate", Clock.UtcNow.AddDays(-1) }
+        };
+        var person = await TestData.CreatePersonAsync(x => x.WithTrn());
+        var trsPerson = context.Persons.Single(x => x.Trn == person.Trn);
+        trsPerson.SetInductionStatus(
+            fromInductionStatus,
+            startDate: null,
+            completedDate: null,
+            exemptionReasonIds: Array.Empty<Guid>(),
+            updatedBy: SystemUser.SystemUserId,
+            Clock.UtcNow.AddDays(-2),
+            out var even);
+        context.AddEventWithoutBroadcast(even!);
+
+        trsPerson.SetInductionStatus(
+            toInductionStatus,
+            startDate: null,
+            completedDate: null,
+            exemptionReasonIds: new[] { InductionExemptionReason.PassedInWalesId },
+            updatedBy: SystemUser.SystemUserId,
+            Clock.UtcNow.AddDays(-2),
+            out var even1);
+        context.AddEventWithoutBroadcast(even1!);
+        await context.SaveChangesAsync();
+
+        // Act
+        await Job.ExecuteAsync(token);
+
+        // Assert
+        var task = ctx.TaskSet.FirstOrDefault(x => x.RegardingObjectId == person.ContactId.ToEntityReference(Contact.EntityLogicalName));
+        Assert.Null(task);
+    }
+
+    [Fact]
+    public async Task Execute_MetadataDoesNotExist_InsertsRow()
+    {
+        // Arrange
+        await DbFixture.DbHelper.ClearDataAsync();
+        var fromInductionStatus = Models.InductionStatus.InProgress;
+        var toInductionStatus = InductionStatus.Exempt;
+        await using var context = await DbFixture.DbHelper.DbContextFactory.CreateDbContextAsync();
+        using var ctx = new DqtCrmServiceContext(OrganizationService);
+        var source = new CancellationTokenSource();
+        var token = source.Token;
+        var person = await TestData.CreatePersonAsync(x => x.WithTrn());
+        var trsPerson = context.Persons.Single(x => x.Trn == person.Trn);
+        trsPerson.SetInductionStatus(
+            fromInductionStatus,
+            startDate: null,
+            completedDate: null,
+            exemptionReasonIds: Array.Empty<Guid>(),
+            updatedBy: SystemUser.SystemUserId,
+            Clock.UtcNow.AddDays(-2),
+            out var even);
+        context.AddEventWithoutBroadcast(even!);
+        var options = Options.Create(new InductionStatusUpdatedSupportJobOptions()
+        {
+            InitialLastRunDate = Clock.UtcNow,
+        });
+        trsPerson.SetInductionStatus(
+            toInductionStatus,
+            startDate: null,
+            completedDate: null,
+            exemptionReasonIds: new[] { InductionExemptionReason.PassedInWalesId },
+            updatedBy: SystemUser.SystemUserId,
+            Clock.UtcNow.AddDays(-2),
+            out var even1);
+        context.AddEventWithoutBroadcast(even1!);
+        await context.SaveChangesAsync();
```

Comment:
Not sure we need any of this if we're just checking the metadata entry has been added